mpfilepicker/src/androidMain/kotlin/com/darkrockstudios/libraries/mpfilepicker/AndroidFilePicker.kt

```diff
@@ -1,18 +1,26 @@
 package com.darkrockstudios.libraries.mpfilepicker
 
+import android.content.Context
 import android.net.Uri
 import android.webkit.MimeTypeMap
 import androidx.activity.compose.rememberLauncherForActivityResult
 import androidx.activity.result.contract.ActivityResultContracts
 import androidx.compose.runtime.Composable
 import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.ui.platform.LocalContext
 import androidx.core.net.toFile
 
 public data class AndroidFile(
 	override val path: String,
 	override val platformFile: Uri,
+	private val context: Context,
 ) : MPFile<Uri> {
-	override suspend fun getFileByteArray(): ByteArray = platformFile.toFile().readBytes()
+	override suspend fun getFileByteArray(): ByteArray {
+		return context.contentResolver
+			.openInputStream(platformFile)
+			.use { inputStream -> inputStream?.readBytes() }
+			?: platformFile.toFile().readBytes()
+	}
```

Comment:
Ideally, we should change the return type of the method to `ByteArray?` and rename the method to `getFileByteArrayOrNull()` to have a clearer API.

That way, we will be able to remove the following code from the current implementation:
```kotlin
?: platformFile.toFile().readBytes(
```
because the existing implementation works for both `content` and `file` schemes.

But I did not change it in this PR since that would mean a breaking change for the existing users. So please feel free to make any such changes (or even let me know the same, I'd be happy to lend a hand) 👍🏻